### ISSUE TITLE
Add more club information to the homepage, especially a description / mission statement

### ISSUE THREAD

**Issue by ChristopherWMM** [MCC-CSClub/mcc-csclub.github.io#36]
As of right now, there is a missing club description on the homepage under the club logo (as seen below).

![missing](https://i.imgur.com/2Zf8Qso.png)

**Comment by JosChavz** [comment]
So I made three concepts, which are all alike.
!First Concept(https://i.imgur.com/dO86NVZ.png)
!Second Concept(https://imgur.com/vKQqCEh.png)
!Third Concept(https://imgur.com/ZhZ9QBS.png)

**Comment by JosChavz** [comment]
Of course. I got the svg files, and yeah I was thinking as well to change colors when it is on hover, perhaps the MCC blue?

**Comment by ChristopherWMM** [comment]
I was going to say we have the links change to the respective colors commonly used on each social media platform. Discord could turn "Bluish" (name of their color), Instagram could turn magenta or something, and so on and so forth.

**Comment by JosChavz** [comment]
I see what you mean, yeah that would actually be a better idea! But what about GitHub and Instagram? Github is normally black and IG is a gradient color (which is possible to add). So the only thing that would be in the problem for hover coloring is GitHub's icon.

**Comment by ChristopherWMM** [comment]
Potentially an off-black, like a charcoal color?

**Comment by ChristopherWMM** [comment]
I love the ideas and collaboration going on here. I made separate issues for each of the suggestions:
- Add an "About us" page #51
- Add social media links to the homepage sidebar #52 
- Add MiraCosta Computer Science department website to the sidebar #53
